internal/server/router/router.go

```diff
@@ -374,7 +374,11 @@ func (a *Router) QueryMeter(w http.ResponseWriter, r *http.Request, meterIDOrSlu
 		}),
 	}
 
-	_ = render.Render(w, r, resp)
+	if r.Header.Get("Accept") == "text/csv" {
```

Comment:
Parsing media type would be safer: https://pkg.go.dev/mime#ParseMediaType

Reply:
Or use a snippet from the render library: https://github.com/go-chi/render/blob/v1.0.3/content_type.go#L67

(We should probably contribute CSV support to that library)